src/core/public/chrome/chrome_service.tsx

```diff
@@ -116,14 +117,21 @@ export class ChromeService {
     const helpSupportUrl$ = new BehaviorSubject<string>(KIBANA_ASK_ELASTIC_LINK);
     const isNavDrawerLocked$ = new BehaviorSubject(localStorage.getItem(IS_LOCKED_KEY) === 'true');
 
+    const formatKbnVersionHeader = () => {
+      const unformattedVersion = this.params.packageInfoVersion;
+      // we assume that the version is valid
+      return `kibana-version-${unformattedVersion.split('.').join('-')}`;
+    };
+
     const headerBanner$ = new BehaviorSubject<ChromeUserBanner | undefined>(undefined);
     const bodyClasses$ = combineLatest([headerBanner$, this.isVisible$!]).pipe(
       map(([headerBanner, isVisible]) => {
-        return [
+        const requiredClasses = [
           'kbnBody',
           headerBanner ? 'kbnBody--hasHeaderBanner' : 'kbnBody--noHeaderBanner',
           isVisible ? 'kbnBody--chromeVisible' : 'kbnBody--chromeHidden',
         ];
+        return [...requiredClasses, formatKbnVersionHeader()];
```

Comment:
NIT: The second array construction (+spread) can be avoided

```ts
return [
   'kbnBody',
   headerBanner ? 'kbnBody--hasHeaderBanner' : 'kbnBody--noHeaderBanner',
   isVisible ? 'kbnBody--chromeVisible' : 'kbnBody--chromeHidden',
   formatKbnVersionHeader()
]
```

Reply:
[Done](https://github.com/elastic/kibana/pull/102443/files#diff-f586b4a13b87b4d9192b54dcd897a24b881f9308a401f4422319c3ffff36044eR134-R138).
I originally had it separate to handle possibly missing package versions.